blazor-wasm/wwwroot/index.html

```diff
@@ -0,0 +1,73 @@
+﻿<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width" />
+    <title>WebApplication1</title>
```

Comment:
Probably need change project's name string everywhere in the code
```suggestion
    <title>BlazorWasmExample</title>
```